Test failure looks real, my guess is it's just a difference between the way couch and sql products render to XML

```
======================================================================
FAIL: test_product_fixture (corehq.apps.commtrack.tests.test_fixture.FixtureTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/commtrack/tests/test_fixture.py", line 119, in test_product_fixture
    ElementTree.tostring(fixture[0])
  File "/mnt/corehq/apps/app_manager/tests/util.py", line 32, in assertXmlEqual
    _check_shared(expected, actual, LXMLOutputChecker(), "xml")
  File "/mnt/corehq/apps/app_manager/tests/util.py", line 126, in _check_shared
    raise AssertionError(message)
AssertionError: XML mismatch
```

Specifically, `cost` and `unit` vary.  The `cost` difference I'd chalk up to type `Decimal` idiosyncracies, but the missing `unit`s could be something more real, like bad sync code.
```
-      <unit/>
+      <unit>icfwaretcbrrcayijniv</unit>
-      <cost>14.85720</cost>
+      <cost>14.8572</cost>
 ```

--- a/corehq/apps/products/fixtures.py
+++ b/corehq/apps/products/fixtures.py
@@ -55,7 +55,7 @@ class ProductFixturesProvider(object):
 
         def get_products():
             return sorted(
-                SQLProduct.by_domain(user.domain),
+                Product.by_domain(user.domain),
                 key=lambda product: product.code
             )
 

